examples/reinforcement_learning/ddpg_pendulum.py

```diff
@@ -0,0 +1,244 @@
+        self.action_low = action_low
+        with self.init_scope():
+            self.l0 = L.Linear(obs_size, n_units)
+            self.l1 = L.Linear(n_units, n_units)
+            self.l2 = L.Linear(n_units, action_size,
+                               initialW=chainer.initializers.HeNormal(1e-3))
+
+    def __call__(self, x):
+        """Compute actions for given observations."""
+        h = F.relu(self.l0(x))
+        h = F.relu(self.l1(h))
+        return squash(self.l2(h),
+                      self.xp.asarray(self.action_low),
+                      self.xp.asarray(self.action_high))
+
+
+def get_action(policy, obs):
+    """Get an action by evaluating a given policy."""
+    obs = policy.xp.asarray(obs[None], dtype=np.float32)
+    with chainer.no_backprop_mode():
+        action = policy(obs).data[0]
+    return chainer.cuda.to_cpu(action)
+
+
+def update(Q, target_Q, policy, target_policy, opt_Q, opt_policy,
+           samples, gamma=0.99):
+    """Update a Q-function and a policy."""
+    xp = Q.xp
+    obs = xp.asarray([sample[0] for sample in samples], dtype=np.float32)
+    action = xp.asarray([sample[1] for sample in samples], dtype=np.float32)
+    reward = xp.asarray([sample[2] for sample in samples], dtype=np.float32)
+    done = xp.asarray([sample[3] for sample in samples], dtype=np.float32)
+    obs_next = xp.asarray([sample[4] for sample in samples], dtype=np.float32)
+
+    def update_Q():
+        # Predicted values: Q(s,a)
+        y = F.squeeze(Q(obs, action), axis=1)
+        # Target values: r + gamma * Q(s,policy(s))
+        with chainer.no_backprop_mode():
+            next_q = F.squeeze(target_Q(obs_next, target_policy(obs_next)),
+                               axis=1)
+            target = reward + gamma * (1 - done) * next_q
+        loss = F.mean_squared_error(y, target)
+        Q.cleargrads()
+        loss.backward()
+        opt_Q.update()
+
+    def update_policy():
+        # Maximize Q(s,policy(s))
+        q = Q(obs, policy(obs))
+        q = q[:]  # Avoid https://github.com/chainer/chainer/issues/2744
+        loss = - F.mean(q)
+        policy.cleargrads()
+        loss.backward()
+        opt_policy.update()
+
+    update_Q()
+    update_policy()
+
+
+def soft_copy_params(source, target, tau):
+    """Make the parameters of a link close to the ones of another link.
+
+    Making tau close to 0 slows the pace of updates, and close to 1 might lead
+    to faster, but more volatile updates.
+    """
+    # Sort params by name
+    source_params = [param for _, param in sorted(source.namedparams())]
+    target_params = [param for _, param in sorted(target.namedparams())]
+    for s, t in zip(source_params, target_params):
+        t.data[:] += tau * (s.data - t.data)
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description='Chainer example: DDPG')
+    parser.add_argument('--env', type=str, default='Pendulum-v0',
+                        help='Name of the OpenAI Gym environment to play')
```

Comment:
"play" -> "interact", "work on" sounds more formal at least to me. Simply removing "to play" is also admissible.